cli/test/fixtures/perf/preload_tester.js

```diff
@@ -12,12 +12,12 @@ document.write('<script src="/perf/level-2.js?warning&delay=500"></script>');
 // delay our preconnect-candidates so that they're not assumed to be working already
 setTimeout(() => {
   // load another origin
-  fetch('http://localhost:10503/preload.html');
+  // fetch('http://localhost:10503/preload.html');
```

Comment:
Oh, I forgot about this. I think I can just drop this line?

Reply:
I think I can drop this, but @paulirish or @adamraine please lmk if you agree

Reply:
Yeah I think it should be fine without